cmd/esc/cli/env.go

```diff
@@ -72,20 +72,27 @@ type environmentRef struct {
 	version     string
 
 	hasAmbiguousPath bool
+	isRelative       bool
 }
 
-func (r *environmentRef) String() string {
+func (r *environmentRef) FullName() string {
```

Comment:
i'd be open to changing this name if there's a better name for this method

Reply:
Yeah I've been wondering if there is a better way to reference this myself. I feel like what we are really trying to communicate is the identifier for the environment which is `{project}/{env}[@{version}]`